task/sast-unicode-check/0.2/sast-unicode-check.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-unicode-check
+spec:
+  description: >-
+    Scans source code for non-printable unicode characters in all text files.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: image-url
+      type: string
+      description: Image URL.
+      default: ""
+    - name: IMAGE_DIGEST
```

Comment:
This change seems inconsistent.  It should have been `image-digest` I guess?

Reply:
indeed , 
missed that one , now should be fixed